test/e2e/tests/identity/account-syncing/multi-srp.spec.ts

```diff
@@ -10,7 +10,7 @@ import {
   UserStorageMockttpController,
   UserStorageMockttpControllerEvents,
 } from '../../../helpers/identity/user-storage/userStorageMockttpController';
-import { loginWithBalanceValidation } from '../../../page-objects/flows/login.flow';
+import { loginWithoutBalanceValidation } from '../../../page-objects/flows/login.flow';
```

Comment:
When the empty state is active, it replaces the regular balance element (`eth-overview__primary-currency`) that `loginWithBalanceValidation` expects to find, causing the test to fail. The empty state appears because the test accounts have a $0 balance on actual mainnets(localhost is excluded from the aggregate balance calculation). Since this test verifies multi-SRP account syncing and token list values using `checkExpectedTokenBalanceIsDisplayed`, not the fiat balance display we are skipping the balance check.